### PR TITLE
Added branded error page and exception logging

### PR DESCRIPTION
**Pull request by ianatesfa**

### CODE REVIEW

File: src/SFA.DAS.Employer.FrontDoor.Web/Infrastructure/RouteNames.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿namespace SFA.DAS.Employer.FrontDoor.Web.Infrastructure
+{
+    public static class RouteNames
+    {
+        public const string ServiceStartDefault = "default";
```
Comment (lazcool) [inline src/SFA.DAS.Employer.FrontDoor.Web/Infrastructure/RouteNames.cs:5]: Could we remove the route names that aren't used?

File: src/SFA.DAS.Employer.FrontDoor.Web/Views/Error/ApplicationError.cshtml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    @*ViewData[nameof(BreadCrumbViewModel)] = new BreadCrumbViewModel {Level = BreadCrumbLevel.Error};*@
+}
+
+<main class="govuk-main-wrapper" id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
+            <p class="govuk-body">Try again later.</p>
+            <p class="govuk-body"><a href="https://help.apprenticeships.education.gov.uk/hc/en-gb#contact-us" class="govuk-link govuk-link--no-visited-state">Contact the Apprenticeship Service</a> if you need help.</p>
```
Comment (lazcool) [inline src/SFA.DAS.Employer.FrontDoor.Web/Views/Error/ApplicationError.cshtml:14]: Not sure if the apprenticeship service is where the user should get help. Question for Becky. (Same for the 404 page.)

File: src/SFA.DAS.Employer.FrontDoor.Web/Infrastructure/RouteNames.cs (hunk 1 above)
Comment (ianatesfa) [reply src/SFA.DAS.Employer.FrontDoor.Web/Infrastructure/RouteNames.cs:5]: Can do, just wasn't sure which we would need so figured would remove the redundant ones later

File: src/SFA.DAS.Employer.FrontDoor.Web/Views/Error/ApplicationError.cshtml (hunk 2 above)
Comment (lazcool) [reply src/SFA.DAS.Employer.FrontDoor.Web/Views/Error/ApplicationError.cshtml:14]: Will add a story for this.

**Pull request merged**